projects/demo/src/modules/directives/mode/mode.template.html

```diff
@@ -7,7 +7,25 @@
     <ng-template pageTab>
         <p i18n>
             <code>tuiMode</code>
-            allows to set style for a scope of interface
+            allows to set style for a scope of interface.
+        </p>
+
+        <p>
+            There are three modes by default:
+            <br />
```

Comment:
Let's use `ol` + `li` for this, `br` is considered bad practice.

Reply:
fixed